src/analysis/inner_product_space/l2_space.lean

```diff
@@ -437,6 +454,22 @@ hilbert_basis.coe_mk hv _
 
 omit hv
 
+-- Note : this should be `b.repr` composed with an identification of `lp (λ i : ι, 𝕜) 2` with
+-- `pi_Lp 2 (λ i : ι, 𝕜)`, but we don't have that yet (July 2022).
+/-- An orthonormal basis is an Hilbert basis. -/
+protected def _root_.orthonormal_basis.to_hilbert_basis [fintype ι] (b : orthonormal_basis ι 𝕜 E) :
+  hilbert_basis ι 𝕜 E :=
+hilbert_basis.mk b.orthonormal
+begin
```

Comment:
Can you make this a little simpler?  This is the "easy" direction and you shouldn't need to invoke the fact that the image is closed.